landscape_setup/concourse.py

```diff
@@ -111,6 +111,38 @@ def create_image_pull_secret(
             image_pull_secret_name=image_pull_secret_name
         )
 
+MITM_CONFIG_CONFIGMAP_NAME = 'mitm-config'
+MITM_SCRIPTS_CONFIGMAP_NAME = 'mitm-scripts'
+MITM_SCRIPT_DIR = '/.mitmscripts'
```

Comment:
please add a brief comment explaining where said images reside.
also: why not use '/mitmscripts' (root directory)?
also: why use hidden directories?